Misc/NEWS.d/next/Library/2018-04-10-20-57-14.bpo-33256.ndHkqu.rst

```diff
@@ -0,0 +1 @@
+Fix display of ``<module>`` call in the html produced by ``cgitb.html()``.
```

Comment:
Please add "Patch by *your name*." And add your name in Misc/ACKS if it was not added before.